ref-implementation/argo-workflows.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
I think the application yaml should go inside the folder representing a package instead of outside